internal/runners/packages/packages.go

```diff
@@ -162,3 +171,57 @@ func getSuggestions(ns model.Namespace, name string) ([]string, error) {
 
 	return suggestions, nil
 }
+
+func installInitial(cfg configurable, out output.Outputer, authentication *authentication.Auth, prompt prompt.Prompter, packageName, packageVersion, languageName string, operation model.Operation, ns model.Namespace) error {
```

Comment:
fwiw not reviewing this function as I don't believe it should exist as per my comment above.